scene/2d/area_2d.cpp

```diff
@@ -566,7 +631,7 @@ void Area2D::_bind_methods() {
 Area2D::Area2D() :
 		CollisionObject2D(PhysicsServer2D::get_singleton()->area_create(), true) {
 	set_gravity(980);
```

Comment:
Off topic for another PR:  I saw this hardcoded value here and EditorSettings used in skeleton_2D (which is in the same folder), so is there any reason to not get this value from the editor settings?

Reply:
Editor settings are not ok in scene nodes because it's editor code, and Skeleton2D should be changed actually. But in this case I think gravity comes from Project settings, so it would be possible.

I'm not totally sure it would be clear enough to change some property default values based on project settings though, because I don't know what should happen if you change the project settings. Should it update any area gravity that uses the default, or keep whatever value for existing nodes? It might be a bit unpredictable to the user.

Reply:
We can continue the discussion in the chat.